src/applications/rated-disabilities/containers/App.jsx

```diff
@@ -7,14 +7,29 @@ import DowntimeNotification, {
 } from '@department-of-veterans-affairs/platform-monitoring/DowntimeNotification';
 import { RequiredLoginView } from '@department-of-veterans-affairs/platform-user/RequiredLoginView';
 import backendServices from '@department-of-veterans-affairs/platform-user/profile/backendServices';
+import { isLoggedIn } from '@department-of-veterans-affairs/platform-user/selectors';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
 
 import AppContent from '../components/AppContent';
 import FeatureFlagsLoaded from '../components/FeatureFlagsLoaded';
 import MVIError from '../components/MVIError';
 import { isLoadingFeatures } from '../selectors';
+import { useBrowserMonitoring } from '../util/datadog-rum/useBrowserMonitoring';
 
 const App = props => {
-  const { featureFlagsLoading, user } = props;
+  const { featureFlagsLoading, loggedIn, user } = props;
+
+  // Add Datadog UX monitoring to the application
+  useBrowserMonitoring({
+    loggedIn,
+    version: '1.0.0',
+    applicationId: 'ec980bd9-5d61-4cf7-88a8-bdbbdb015059',
+    clientToken: 'pub7162d18113213637d731bd1ae8a0abf0',
+    service: 'benefits-rated-disabilities',
+    sessionSampleRate: environment.vspEnvironment() !== 'production' ? 100 : 10,
```

Comment:
I put these here so they are easier to find. I think it makes sense to put all of the unique configuration stuff here so that it's easy to get a grasp of what's going on without having to go between multiple files